libraries/AP_VideoTX/AP_VideoTX_config.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include <AP_HAL/AP_HAL_Boards.h>
+#include <AP_OSD/AP_OSD.h>
+
+#ifndef AP_VIDEOTX_ENABLED
+#define AP_VIDEOTX_ENABLED 1
+#endif
+
+#ifndef AP_TRAMP_ENABLED
+#define AP_TRAMP_ENABLED AP_VIDEOTX_ENABLED && OSD_ENABLED && BOARD_FLASH_SIZE>1024
```

Comment:
We should change this - people often want trap on 1mb boards

Reply:
> We should change this - people often want trap on 1mb boards

Unrelated change, yes?